src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/GetAzureApplicationGatewayRedirectConfigurationCommand.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmApplicationGatewayRequestRoutingRule"),
+        OutputType(typeof(PSApplicationGatewayRedirectConfiguration), typeof(IEnumerable<PSApplicationGatewayRedirectConfiguration>))]
+    public class GetAzureApplicationGatewayRedirectConfigurationCommand : NetworkBaseCmdlet
```

Comment:
@akshaysngupta I'm not sure I understand what you're trying to do with this cmdlet. It seems like it's just returning the property of the application gateway object that you are providing it, is that correct?